src/Core/Services/Implementations/StripePaymentService.cs

```diff
@@ -427,7 +436,15 @@ private async Task ChangeOrganizationSponsorship(Organization org, OrganizationS
                 Source = stipeCustomerSourceToken,
                 InvoiceSettings = new Stripe.CustomerInvoiceSettingsOptions
                 {
-                    DefaultPaymentMethod = stipeCustomerPaymentMethodId
+                    DefaultPaymentMethod = stipeCustomerPaymentMethodId,
+                    CustomFields = new List<Stripe.CustomerInvoiceSettingsCustomFieldOptions>
+                    {
+                        new Stripe.CustomerInvoiceSettingsCustomFieldOptions()
+                        {
+                            Name = "User",
+                            Value = user.Name,
```

Comment:
I think it's best to use the `ISubscriber` interface method, so that we have a consistent interface for this value.